install/connector/bin/ts.jte

```diff
@@ -105,6 +105,7 @@ javatest.timeout.factor=1.0
 #
 ########################################################################
 connector.home=/sun/glassfish4/glassfish
```

Comment:
@gurunrao could you please also update the ts.jte files for JDK11?  
List of new ts.jte.jdk11 files that should work for JDK11:
```
./install/servlet/bin/ts.jte.jdk11
./install/jacc/bin/ts.jte.jdk11
./install/jta/bin/ts.jte.jdk11
./install/websocket/bin/ts.jte.jdk11
./install/securityapi/bin/ts.jte.jdk11
./install/saaj/bin/ts.jte.jdk11
./install/jsonb/bin/ts.jte.jdk11
./install/caj/bin/ts.jte.jdk11
./install/jsp/bin/ts.jte.jdk11
./install/jaspic/bin/ts.jte.jdk11
./install/jms/bin/ts.jte.jdk11
./install/jpa/bin/ts.jte.jdk11
./install/jaxrs/bin/ts.jte.jdk11
./install/jsonp/bin/ts.jte.jdk11
./install/el/bin/ts.jte.jdk11
./install/jstl/bin/ts.jte.jdk11
./install/concurrency/bin/ts.jte.jdk11
./install/jaxws/bin/ts.jte.jdk11
./install/jsf/bin/ts.jte.jdk11
./install/connector/bin/ts.jte.jdk11
```

Reply:
sure @scottmarlow , since now ts.jte.jdk11 changes are merged. I can add my changes to both jkd8 and jdk11 ts.jte files.

Reply:
@scottmarlow i have updated ts.jte.jdk11 with relevant changes for this PR.